### PR TITLE
Feature/38 connect entities  to backend

### LINKED ISSUE #38: feat: Connect ListEntities to Backend
## Feature Description
Connect ListEntities to Backend will establish communication between the ListEntities component and the backend server, enabling the component to fetch and display data from the server dynamically.
## Motivation
This feature is essential to provide real-time or updated information to users without relying on static data. By connectinga all the Entities to the backend, we ensure that the displayed data remains current and accurate, enhancing the overall user experience.

## Acceptance Criteria
- [ ] [Criteria 1: The Entities  successfully fetches data from the backend API.

### PR DESCRIPTION
**Pull request by Manuelgruiz**
# Description
Implemented a new feature that connects beneficiaries and interventions with backend.

# Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Comment by Manuelgruiz** [comment]
I've established the connection to beneficiaries, but I've changed an attribute to date and haven't been able to connect intervention because I don't have the GET request in the backend.

**Review by JaviGarc1a — CHANGES_REQUESTED** [review]
I can't run the backend, but you should add error handling in both fetches. In the catch you add the following text "No hay datos" and you wrap fetch with Suspense component with a fallback. Example fallback:
```html
<div>Cargando...</div>
```

You ask Álvaro about the vercel problem and try to fix it.

File: src/app/interventions/fetchIntervention.js
Comment (sebasruii) [inline src/app/interventions/fetchIntervention.js:1]: This fetcch still uses mockAPI, it should be connected to backend

**Review by sebasruii — CHANGES_REQUESTED** [review]
image
Fetch interventions is still connected to MockAPI

**Review by sebasruii — APPROVED** [review]
Good job!

**Comment by sebasruii** [comment]
Please, fix the conflicts and I will merge

**Review by sebasruii — APPROVED** [review]

**Comment by sebasruii** [comment]
Since @JaviGarc1a  is not available this week, I'll be taking over and merge this PR

**Pull request merged**